db/migrate/20181215211701_create_items.rb

```diff
@@ -6,7 +6,7 @@ def change
       t.integer :inventory
       t.string :description
       t.integer :current_price
-      t.boolean :enabled
+      t.boolean :enabled, default: true
```

Comment:
Little bitty one here but is this going to bite us later on? I guess we can cross that bridge when we get there :)

Reply:
@danbriechle I don't believe it will, because we'll have the ability to disable the item. Also here's the snippet from the user story ```When I submit valid information and save the form
I am taken back to my items page
I see a flash message indicating my new item is saved
I see the new item on the page, and it is enabled and available for sale``` 
So it looks like items are supposed to be enabled by default per this story.

Reply:
sounds good to me

Reply:
I think this confusion came from something we heard in class. Ian said that in a usual production environment, items are usually created an NOT enabled, so that the merchant or site planner can get it properly created and set up before going live. He also specified that we are not going to do this in our Little Shop project.

Reply:
@saleratus - oh yeah that's where we heard that - you're totally right!